index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Assignment sollution for Module 2</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Alice&display=swap" rel="stylesheet">
+   </head>
+   
+<body>
```

Comment:
Fix indentation -> https://google.github.io/styleguide/htmlcssguide.html#Indentation
Everything inside the body should be indented by one level.

Reply:
Mnolo!